adafruit_button/button_base.py

```diff
@@ -0,0 +1,152 @@
+# SPDX-FileCopyrightText: 2022 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`adafruit_button.button`
+================================================================================
+
+UI Buttons for displayio
+
+
+* Author(s): Limor Fried
+
+Implementation Notes
+--------------------
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+from adafruit_display_text.bitmap_label import Label
+from displayio import Group
+
+
+def _check_color(color):
+    # if a tuple is supplied, convert it to a RGB number
+    if isinstance(color, tuple):
+        r, g, b = color
+        return int((r << 16) + (g << 8) + (b & 0xFF))
+    return color
+
+
+class ButtonBase(Group):
+    # pylint: disable=too-many-instance-attributes
+    """Superclass for creating UI buttons for ``displayio``.
+
+    :param x: The x position of the button.
+    :param y: The y position of the button.
+    :param width: The width of the button in tiles.
+    :param height: The height of the button in tiles.
+    :param label: The text that appears inside the button. Defaults to not displaying the label.
+    :param label_font: The button label font.
+    :param label_color: The color of the button label text. Defaults to 0x0.
+    """
```

Comment:
`selected_label` and `name` are not included in the documentation

Reply:
added both with latest commit